obspy/imaging/cm.py

```diff
@@ -24,42 +31,67 @@
 import numpy as np
 from matplotlib.cm import get_cmap
 from matplotlib.colors import LinearSegmentedColormap
+from matplotlib.colors import ListedColormap
 
 
-def _get_cmap(name, lut=None, reverse=False):
+def _get_cmap(name, kind='segmented', lut=None, reverse=False):
     """
     Load a :class:`~matplotlib.colors.LinearSegmentedColormap` from
     `segmentdata` dictionary saved as numpy compressed binary data.
 
     :type name: str
     :param name: Name of colormap to load, same as filename in
-        `obspy/imaging/data` without `.npz` file suffix.
+        `obspy/imaging/data` without `.npz or .npy` file suffix.
+    :type kind: str
```

Comment:
I think you can also just use the `glob` module and search for `NAME.np?`. If two are found, just raise an exception - that should not happen in any case. Otherwise this looks pretty ready to merge.